src/overlays/actors/ovl_En_Fall/z_en_fall.c

```diff
@@ -762,62 +765,62 @@ void EnFall_LodMoon_DrawWithLerp(Actor* thisx, GlobalContext* globalCtx) {
     EnFall_LodMoon_Draw(thisx, globalCtx);
 }
 
-void EnFall_FireBall_Draw(Actor* thisx, GlobalContext* globalCtx) {
+void EnFall_Fireball_Draw(Actor* thisx, GlobalContext* globalCtx) {
     s32 pad;
     EnFall* this = THIS;
     u32 gameplayFrames = globalCtx->gameplayFrames;
 
     OPEN_DISPS(globalCtx->state.gfxCtx);
     gSPMatrix(POLY_XLU_DISP++, Matrix_NewMtx(globalCtx->state.gfxCtx), G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
     func_8012C2DC(globalCtx->state.gfxCtx);
-    this->fireBallYTexScroll1 += (s32)(4.0f + (this->fireBallIntensity * 12.0f));
-    this->fireBallYTexScroll2 += (s32)(2.0f + (this->fireBallIntensity * 6.0f));
+    this->fireballYTexScroll1 += (s32)(4.0f + (this->fireballIntensity * 12.0f));
+    this->fireballYTexScroll2 += (s32)(2.0f + (this->fireballIntensity * 6.0f));
     gDPSetPrimColor(POLY_XLU_DISP++, 0, 0x80,
-                    (s32)(((1.0f - this->fireBallIntensity) * 160.0f) + (255.0f * this->fireBallIntensity)),
-                    (s32)((70.0f * (1.0f - this->fireBallIntensity)) + (255.0f * this->fireBallIntensity)),
-                    (s32)(70.0f * (1.0f - this->fireBallIntensity)), 255);
+                    (s32)(((1.0f - this->fireballIntensity) * 160.0f) + (255.0f * this->fireballIntensity)),
+                    (s32)((70.0f * (1.0f - this->fireballIntensity)) + (255.0f * this->fireballIntensity)),
+                    (s32)(70.0f * (1.0f - this->fireballIntensity)), 255);
     gDPSetEnvColor(
-        POLY_XLU_DISP++, (s32)(((1.0f - this->fireBallIntensity) * 50.0f) + (200.0f * this->fireBallIntensity)),
-        (s32)(20.0f * (1.0f - this->fireBallIntensity)), (s32)(20.0f * (1.0f - this->fireBallIntensity)), 255);
+        POLY_XLU_DISP++, (s32)(((1.0f - this->fireballIntensity) * 50.0f) + (200.0f * this->fireballIntensity)),
+        (s32)(20.0f * (1.0f - this->fireballIntensity)), (s32)(20.0f * (1.0f - this->fireballIntensity)), 255);
 
     // Glowing sphere of fire
```

Comment:
I think it's fairer to say that these are used on this particular component: currently it sounds like this is the whole thing, which of course it is not.

Reply:
Can you clarify what you mean by "these"? I'm not sure I'm quite following what you mean here.

Reply:
I mean the texscrolls: they're part of drawing these bits, but not all of them.

Reply:
Oh, I think I get what you mean. You're basically saying that these comments make it look like the TwoTexScrolls are actually *drawing* the "glowing sphere of fire", when it's actually just part of drawing it, right? Let me update the comments to be a little more specific.

Reply:
Yeah, it could be interpreted as being the whole drawing operation rather than just part of it, exactly.